activerecord/lib/arel/collectors/sql_string.rb

```diff
@@ -12,7 +12,7 @@ def initialize(*)
         @bind_index = 1
       end
 
-      def add_bind(bind)
+      def add_bind(bind, &)
         self << yield(@bind_index)
```

Comment:
Actually, this one isn't strictly necessary, because the compiler should be able to see the `yield`.

Reply:
I suppose it doesn't hurt to make the signature more explicit.

Reply:
I didn't actually mean to add this here and I intentionally left it out for another method that also yields. I added the block there as well